includes/api/class-wc-rest-order-notes-controller.php

```diff
@@ -83,6 +83,87 @@ public function get_items( $request ) {
 	}
 
 	/**
+	 * Prepare a single order note output for response.
+	 *
+	 * @param WP_Comment $note Order note object.
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response $response Response data.
+	 */
+	public function prepare_item_for_response( $note, $request ) {
+		$data = array(
+			'id'               => (int) $note->comment_ID,
+			'date_created'     => wc_rest_prepare_date_response( $note->comment_date ),
+			'date_created_gmt' => wc_rest_prepare_date_response( $note->comment_date_gmt ),
+			'note'             => $note->comment_content,
+			'customer_note'    => (bool) get_comment_meta( $note->comment_ID, 'is_customer_note', true ),
```

Comment:
We save in the data base only `0` and `1`, but sorry, I did not changed it in this Pull REquest, just included `date_created_gmt`...